engine/src/main/java/org/entando/entando/aps/system/common/model/DeleteDto.java

```diff
@@ -0,0 +1,31 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package org.entando.entando.aps.system.common.model;
+
+/**
+ *
+ * @author paddeo
+ */
+public class DeleteDto {
```

Comment:
I don't think we want to do this. Generally, not a good thing to have objects that represent verbs. If we need to return a spares object either return a Map with the few things in it you need or a sparsely populated instance of the thing being deleted.